README.md

```diff
@@ -1,70 +1,76 @@
-# Getting Started with Create React App
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
```

Comment:
- [ ] You have correctly used the recommended README template. But it is still a dummy file. Kindly update your README with the necessary information regarding this project.